Core/include/Acts/TrackFitting/Chi2Fitter.hpp

```diff
@@ -597,11 +608,12 @@ class Chi2Fitter {
   /// the fit.
   ///
   /// @return the output as an output track
-  template <typename source_link_iterator_t, typename start_parameters_t,
+  template <typename source_link_iterator_t,
             typename parameters_t = BoundTrackParameters>
```

Comment:
If we still want to fix our type to `BoundTrackParameters` for now, I think we can remove `parameters_t` for now as well?